src/containers/Cluster/Cluster.tsx

```diff
@@ -101,67 +102,127 @@ function Cluster({
+                                to={path}
+                                key={id}
+                                onClick={() => {
+                                    dispatch(updateDefaultClusterTab(id));
+                                }}
+                            >
                                 {node}
                             </InternalLink>
                         );
                     }}
                 />
             </div>
 
-            <div>{renderTab()}</div>
+            <div>
+                <Switch>
+                    <Route
+                        path={
+                            getLocationObjectFromHref(getClusterPath(clusterTabsIds.overview))
+                                .pathname
+                        }
+                    >
+                        <ClusterInfo
+                            cluster={cluster}
+                            groupsStats={groupsStats}
+                            versionsValues={versionsValues}
+                            loading={infoLoading}
+                            error={clusterError}
+                            additionalClusterProps={additionalClusterProps}
+                        />
+                    </Route>
+                    <Route
+                        path={
+                            getLocationObjectFromHref(getClusterPath(clusterTabsIds.tenants))
+                                .pathname
+                        }
+                    >
+                        <Tenants additionalTenantsProps={additionalTenantsProps} />
+                    </Route>
+                    <Route
+                        path={
+                            getLocationObjectFromHref(getClusterPath(clusterTabsIds.nodes)).pathname
+                        }
+                    >
+                        <NodesWrapper
+                            parentContainer={container.current}
+                            additionalNodesProps={additionalNodesProps}
+                        />
+                    </Route>
+                    <Route
+                        path={
+                            getLocationObjectFromHref(getClusterPath(clusterTabsIds.storage))
+                                .pathname
+                        }
+                    >
+                        <StorageWrapper
+                            parentContainer={container.current}
+                            additionalNodesProps={additionalNodesProps}
+                        />
+                    </Route>
+                    <Route
+                        path={
+                            getLocationObjectFromHref(getClusterPath(clusterTabsIds.versions))
+                                .pathname
+                        }
+                    >
+                        <Versions versionToColor={versionToColor} />
+                    </Route>
+                    <Redirect to={getLocationObjectFromHref(getClusterPath(defaultTab))} />
```

Comment:
Why do you need `Redirect` here? `activeTab` is always a `ClusterTab` and it equals to `defaultTab`, if it didn't matched before, something is wrong, `Redirect` won't help

Reply:
we go to the cluster page (`/cluster`) and it redirects to the default tab.